html-block-and-inline/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <style>
+
+  </style>
+</head>
+<body>
+  <div>
+    <h1> Blue Americana Snuggie &reg; Blanket</h1>
+    <h2> by SnuggieStore</h2>
+    <p>
+      Price: <strike>$19.99</strike> $14.99 <br/>
+      <br />
+      Part of our new Americana collection, theis <i>beautiful</i> blue is already a customer favorite!<br />
+      <br />
+      Enjoy complete <b>Freedom of Movement!</b><br />
+      <br />
+      Does not slip and slide like a regular blanket and keeps you warm from head to toe.<br />
+    </p>
```

Comment:
Each line should be in their own `<p>` elements. We typically want to avoid using `<br>` tags.